src/Umbraco.Web.Website/Extensions/HtmlHelperRenderExtensions.cs

```diff
@@ -33,34 +35,20 @@ namespace Umbraco.Extensions
     public static class HtmlHelperRenderExtensions
     {
         private static T GetRequiredService<T>(IHtmlHelper htmlHelper)
-        {
-            return GetRequiredService<T>(htmlHelper.ViewContext);
-        }
+            => GetRequiredService<T>(htmlHelper.ViewContext);
 
         private static T GetRequiredService<T>(ViewContext viewContext)
-        {
-            return viewContext.HttpContext.RequestServices.GetRequiredService<T>();
-        }
+            => viewContext.HttpContext.RequestServices.GetRequiredService<T>();
 
         /// <summary>
         /// Renders the markup for the profiler
         /// </summary>
-        /// <param name="helper"></param>
-        /// <returns></returns>
         public static IHtmlContent RenderProfiler(this IHtmlHelper helper)
-        {
-            return new HtmlString(GetRequiredService<IProfilerHtml>(helper).Render());
-        }
+            => new HtmlString(GetRequiredService<IProfilerHtml>(helper).Render());
 
         /// <summary>
         /// Renders a partial view that is found in the specified area
         /// </summary>
-        /// <param name="helper"></param>
-        /// <param name="partial"></param>
-        /// <param name="area"></param>
-        /// <param name="model"></param>
-        /// <param name="viewData"></param>
-        /// <returns></returns>
         public static IHtmlContent AreaPartial(this IHtmlHelper helper, string partial, string area, object model = null, ViewDataDictionary viewData = null)
         {
             var originalArea = helper.ViewContext.RouteData.DataTokens["area"];
```

Comment:
I think we should consider having `"area"` as a constant. We use it a magic string all over

Reply:
Yep, will adjust these constants. But good find, since I'm unsure this method is even necessary or works this way anymore. I don't think 'area' in DataTokens is a thing but I'll need to test these things. This also reminds me that I need to test areas with surface controllers, etc....